src/ServiceControl.Transports/TransportCustomization.cs

```diff
@@ -2,22 +2,60 @@
+        protected abstract void CustomizeTransportSpecificServiceControlEndpointSettings(EndpointConfiguration endpointConfiguration, TransportSettings transportSettings);
 
-        public abstract void CustomizeServiceControlEndpoint(EndpointConfiguration endpointConfiguration, TransportSettings transportSettings);
+        protected abstract void CustomizeTransportSpecificSendOnlyEndpointSettings(EndpointConfiguration endpointConfiguration, TransportSettings transportSettings);
 
-        public abstract void CustomizeMonitoringEndpoint(EndpointConfiguration endpointConfiguration, TransportSettings transportSettings);
+        protected abstract void CustomizeTransportSpecificMonitoringEndpointSettings(EndpointConfiguration endpointConfiguration, TransportSettings transportSettings);
 
         public abstract void CustomizeForReturnToSenderIngestion(RawEndpointConfiguration endpointConfiguration, TransportSettings transportSettings);
 
+        public void CustomizeServiceControlEndpoint(EndpointConfiguration endpointConfiguration, TransportSettings transportSettings)
+        {
+            ConfigureDefaultEndpointSettings(endpointConfiguration, transportSettings);
+            CustomizeTransportSpecificServiceControlEndpointSettings(endpointConfiguration, transportSettings);
+        }
+
+        public void CustomizeSendOnlyEndpoint(EndpointConfiguration endpointConfiguration, TransportSettings transportSettings)
+        {
+            ConfigureDefaultEndpointSettings(endpointConfiguration, transportSettings);
+            CustomizeTransportSpecificSendOnlyEndpointSettings(endpointConfiguration, transportSettings);
+
+            endpointConfiguration.SendOnly();
+
+            //DisablePublishing API is available only on TransportExtensions for transports that implement IMessageDrivenPubSub so we need to set settings directly
+            endpointConfiguration.GetSettings().Set("NServiceBus.PublishSubscribe.EnablePublishing", false);
+        }
+
+        public void CustomizeMonitoringEndpoint(EndpointConfiguration endpointConfiguration, TransportSettings transportSettings)
+        {
+            ConfigureDefaultEndpointSettings(endpointConfiguration, transportSettings);
+            CustomizeTransportSpecificMonitoringEndpointSettings(endpointConfiguration, transportSettings);
+        }
+
+        protected void ConfigureDefaultEndpointSettings(EndpointConfiguration endpointConfiguration, TransportSettings transportSettings)
+        {
+            endpointConfiguration.DisableFeature<Audit>();
+            endpointConfiguration.DisableFeature<AutoSubscribe>();
+            endpointConfiguration.DisableFeature<Outbox>();
+            endpointConfiguration.DisableFeature<Sagas>();
+            endpointConfiguration.DisableFeature<TimeoutManager>();
+            endpointConfiguration.SendFailedMessagesTo(transportSettings.ErrorQueue);
+
+            endpointConfiguration.UsePersistence<InMemoryPersistence>();
```

Comment:
@WilliamBZA didn't we say to either remove this or move it out to the monitoring instance? (since the main instance sets this to the cached raven persister)

Reply:
Removed the call completely. Doesn't seem to be needed anywhere (confirmed with smoke testing too)

Reply:
Yes, that was my hunch to